brainstormforce/astra
# AST-2124 - Astra Pro upgrade to pro implementations

### Description
- Upgrade to Pro notices implementation for Astra Pro
- 1. Customizer configurations in sections
- 2. Header/Footer Builder area
- 3. Editor meta setting box
- 4. Notice on WooCommerce pages
- 5. Check pro post types visuals (Custom Layouts, Page Header)
- Following filter to disable upgrade notices
```
add_filter( 'astra_display_pro_upgrade_notices', '__return_false' );
```

### Screenshots
1. Customizer - https://share.bsf.io/d5unemON
2. Customizer - https://share.bsf.io/Jru87KoY
3. Editor - https://share.bsf.io/rRuQL4Q1
4. On WooCommerce settings pages - https://share.bsf.io/nOubmqQv
5. Pro post types - https://share.bsf.io/nOubmq24

### Types of changes
- Upgrade to Pro notices (non-breakable change)

### How has this been tested?
- Check all the instances mentioned above
- Check via filter to disable notice
- Check UI, wordings (if needs to improve somewhere)

### Checklist:
